<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArmInfo</class>
 <widget class="QWidget" name="ArmInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lb_peopleIcon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>人物头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_armInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>280</width>
     <height>110</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_armProper">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>武器属性</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_attack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>攻击</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_attackValue">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bisha">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>必杀</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bishaValue">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mingzhong">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>命中</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mingzhongValue">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_huibi">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>回避</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_huibiValue">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>lb_peopleIcon</zorder>
  <zorder>lb_armInfo</zorder>
  <zorder>lb_armProper</zorder>
  <zorder>lb_attack</zorder>
  <zorder>lb_attackValue</zorder>
  <zorder>lb_bisha</zorder>
  <zorder>lb_bishaValue</zorder>
  <zorder>lb_mingzhong</zorder>
  <zorder>lb_mingzhongValue</zorder>
  <zorder>lb_huibi</zorder>
  <zorder>lb_huibiValue</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
